### PR TITLE
OSD-24269: Add cluster login support via OHSS card 

### PR DESCRIPTION
**Pull request by samanthajayasinghe**
### What type of PR is this?

feature

### What this PR does / Why we need it?
Login cluster via JIRA ID 

### Which Jira/Github issue(s) does this PR fix?

_Resolves #_

#### Guidelines
- If it's a new sub-command or new function to an existing sub-command, please cover at least 50% of the code
- If it's a bug fix for an existing sub-command, please cover 70% of the code 
 
#### Test coverage checks  
- [x] Added unit tests
- [ ] Created jira card to add unit test
- [ ] This PR may not need unit tests

### Pre-checks (if applicable)
- [x] Ran unit tests locally
- [x] Validated the changes in a cluster
- [ ] Included documentation changes with PR

### CODE REVIEW

File: pkg/elevate/England
Comment (feichashao) [inline pkg/elevate/England:1]: what is this file for?

**Comment by feichashao** [comment]
I hit an error,
```
 ./ocm-backplane login --ohss OHSS-xxxxx 
ERRO[0008] issue OHSS-35468 is not belongs to OHSS project 
```
let me send you the OHSS ticket number offline.

File: pkg/elevate/England
Comment (samanthajayasinghe) [reply pkg/elevate/England:1]: unit test leftovers :)

**Pull request merged**